src/python/pants/backend/jvm/tasks/jvm_compile/jvm_compile_global_strategy.py

```diff
@@ -369,7 +365,7 @@ def post_process_cached_vts(self, cached_vts):
 
     Class files are retrieved directly into their final locations in the global classes dir.
     """
-    self._ensure_analysis_tmpdir()
+    self.ensure_analysis_tmpdir()
```

Comment:
I know this is pre-existing but  I don't understand why we call this again.   Leave it but maybe put a comment so we can talk about it during the public review?